MotionPhotoReader/app/src/androidTest/java/com/google/android/libraries/motionphotoreader/MotionPhotoReaderTest.java

```diff
@@ -145,83 +156,43 @@ public void getCurrentTimestamp_seekTo_isCorrect() throws IOException, XMPExcept
                     flag);
             currentTimestampUs = newTimestampUs;
         }
-        cleanup.add(reader::close);
     }
 
     @Test
-    public void hasNextFrame_atBeginningOfVideo_returnsTrue() throws IOException, XMPException {
-        MotionPhotoReader reader = MotionPhotoReader.open(
-                ResourceFetcher.fetchAssetFile(context, filename, "test_photo", ".jpg"),
-                null
-        );
+    public void hasNextFrame_atBeginningOfVideo_returnsTrue() {
         boolean flag = reader.hasNextFrame();
         assertTrue("No next frame", flag);
```

Comment:
nit: "flag" temp variable is not necessary. Reming it would make the code cleaner. (same for a bunch of other places)